src/menus/ClipMenus.cpp

```diff
@@ -110,10 +110,10 @@ double AdjustForFindingStartTimes(
 {
    auto q = std::find_if(clips.begin(), clips.end(),
       [&] (const WaveClip* const& clip) {
-         return clip->GetEndTime() == time; });
```

Comment:
Comments above this function, and above the next function, still mention the old method names `GetStartTime, GetEndTime`.  Update the comments.